modules/util/portal-tools-rest-builder/src/test/resources/com/liferay/portal/tools/rest/builder/dependencies/rest-openapi.yaml

```diff
@@ -76,6 +76,26 @@ components:
                     type: array
             title: TestArray
             type: object
+        TestWithJsonMapAttribute:
+            description:
+                Test Component to test the generation of getValue method on DTOs when a Json Map attribute is present.
+            properties:
+                description:
+                    type: string
+                name:
+                    type: string
+                properties:
```

Comment:
It's weird one calling `properties` and the other `properties2`. If you are testing both properties, you must follow the nomenclature: `property1` and `porperty2`.

Reply:
I would say `properties1` and `properties2`

Reply:
Ok! I'm on it!